src/Controller/KeyListBuilder.php

```diff
@@ -33,4 +33,14 @@ public function buildRow(EntityInterface $entity) {
     return $row + parent::buildRow($entity);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function render() {
+    $entities = $this->load();
+    $build = parent::render();
+    $build['table']['#empty'] = t('No keys are available. <a href="@link">Add a key</a>.', array('@link' => \Drupal::url('entity.key.add_form')));
```

Comment:
Does this output a real link these days with all the SafeMarkup changes in Twig? That could be a good assertion for integration/functional test.